### DISCUSSION TITLE
教程太复杂了，N卡到底该怎么设置？

### DISCUSSION THREAD

**Discussion by ckx382** [hooke007/mpv_PlayKit#439] [mpv 通用]
RTX系列的N卡，怎么简单上手啊，滤镜和着色器选哪个？硬件解码选哪个？

**Comment by hooke007** [comment]
不做任何设置就能直接用

**Comment by hong-tm** [comment]
### 来抄个作业，目前是mpv-lazy-2024V0，主要是看动漫为主，有没有适合的滤镜配置推荐，主要使用24寸1080 144hz(主力), 笔记本 2.5k 165 hz。

### 目前配置
![image](https://github.com/hooke007/MPV_lazy/assets/51999870/7abbb810-358d-4ad0-8b0a-da27fbe7c367)
![image](https://github.com/hooke007/MPV_lazy/assets/51999870/99081291-0a69-4f85-9acd-77d004dec1c6)

### mpv配置
![image](https://github.com/hooke007/MPV_lazy/assets/51999870/f711c2ea-cc91-4bfc-be67-8ae0fb23df00)

**Reply by AcerolaShinobu** [reply]
1080P看动画就用 RIFE v4.15 Lite 补个3倍72帧吧

**Reply by AcerolaShinobu** [reply]
或者试试先降分辨率到720p再用ESRGAN_NV放大，对老动画有奇效，1080p的日本动画一般都由原生分辨率720p或800左右拉伸上来的，如果喜欢锐利点的画面可以先降回720p再用ESRGAN_NV放大。
clip = k7f.FMT_CTRL(clip, h_max=720, fmt_pix=0)
clip = k7f.ESRGAN_NV(clip, lt_hd=True, model=5005, gpu=0, gpu_t=2, st_eng=St_Eng, ws_size=0)
clip = k7f.FMT_CTRL(clip, h_max=1080, fmt_pix=0)
clip = k7f.RIFE_NV(clip, lt_d2k=False, model=4151, ext_proc=Ext_Proc, t_tta=False, fps_in=container_fps, fps_num=2, fps_den=1, sc_mode=0, gpu=0, gpu_t=2, st_eng=St_Eng, ws_size=0)

**Reply by AcerolaShinobu** [reply]
<img width="1920" alt="0000" src="https://github.com/hooke007/MPV_lazy/assets/38858821/8627a20b-38bc-40d4-9a3a-5d54b5dbb992">
<img width="1920" alt="1111" src="https://github.com/hooke007/MPV_lazy/assets/38858821/4c08e07f-aad7-4374-b5af-c19ebbaa8368">
<img width="1920" alt="2222" src="https://github.com/hooke007/MPV_lazy/assets/38858821/23e2aab4-2687-43ca-8ca0-0daccf9e2a3d">

上图是仅开启 Anime4K_Clamp_Highlights.glsl + Anime4K_Restore_CNN_VL + Anime4K_Upscale_CNN_x2_L 的效果，
中图是先降分辨率到720p再用ESRGAN_NV放大后再叠加 Anime4K 着色器的效果，
下图是先降分辨率到540p再用ESRGAN_NV放大到4K后再叠加 Anime4K 着色器的效果，
片源是[philosophy-raws] 压制的夏娜第一季，1080p。我比较喜欢清晰的画面，老动画即使是蓝光片源也很模糊，比较适合先降分辨率再放大处理。

**Reply by hong-tm** [reply]
>1080P看动画就用 RIFE v4.15 Lite 补个3倍72帧吧

我很少使用补帧

**Reply by hong-tm** [reply]
> 或者试试先降分辨率到720p再用ESRGAN_NV放大，对老动画有奇效，1080p的日本动画一般都由原生分辨率720p或800左右拉伸上来的，如果喜欢锐利点的画面可以先降回720p再用ESRGAN_NV放大。

看起来蛮有意思的

**Reply by hong-tm** [reply]
> 上图是仅开启 Anime4K_Clamp_Highlights.glsl + Anime4K_Restore_CNN_VL + Anime4K_Upscale_CNN_x2_L 的效果，
中图是先降分辨率到720p再用ESRGAN_NV放大后再叠加 Anime4K 着色器的效果，
下图是先降分辨率到540p再用ESRGAN_NV放大到4K后再叠加 Anime4K 着色器的效果，

三张对比蛮大的，真的差距如此之大🙀

**Reply by 2607627233** [reply]
glsl-shader= ~~/shaders/Anime4K/Anime4K_Thin_HQ.glsl
glsl-shader= ~~/shaders/Anime4K/Anime4K_Restore_CNN_Soft_VL.glsl
glsl-shader= ~~/shaders/Anime4K/Anime4K_Clamp_Highlights.glsl

大家搭配都差不多啊，我是用这三个

**Reply by hong-tm** [reply]
## Gan 比较好吗？

![image](https://github.com/hooke007/MPV_lazy/assets/51999870/f33758f8-ba52-4da7-9c32-8a24653f460d)

**Comment by AcerolaShinobu** [comment]
MPC-HC + MpcVideoRenderer + SVP 最简单省事，还能调用N卡的 RTX Video Super Resolution。

**Reply by ckx382** [reply]
哪里下载呀，再加补帧，rtx2080显卡
